.travis.yml

```diff
@@ -17,19 +16,13 @@ services:
   - postgresql
 
 env:
-  - DJANGO="2.0"
-  - DJANGO="2.1"
   - DJANGO="2.2"
   - DJANGO="3.0"
   - DJANGO="3.1"
+  - DJANGO="3.2"
```

Comment:
The Travis CI is not working properly because I've run out of credits; I'll try to see if I can restart the CI to see if it's working. I'm planning on moving to GitHub Actions once I find the chance to finish off #173